### ISSUE TITLE
[Component Update] A11Y: When "high contrast mode" is enabled on mobile, the radio button (border and indicator) must be displayed in black over white in light mode. No change required in dark mode.

### ISSUE THREAD

**Issue by ludovic35** [Orange-OpenSource/ouds-ios#571]
### Description

A11Y: When "high contrast mode" is enabled on mobile, the radio button (border and indicator) must be displayed in black over white in light mode. No change required in dark mode.

### Types of change

- [ ] Add new component
- [ ] Update existing component

### Checklist for Core Team

- [ ] Tokens have been updated
- [ ] Documentations have been updated
- [ ] Unit tests have been updated
- [ ] UI tests have been updated
- [ ] Design System Toolbox has been updated
- [ ] Retrocompatibility has been checked, or broken API have been flagged as deprecated with a workaround

**Comment by pylapp** [comment]
_on hold_ until we talked all together with the design team about the high contrast mode

**Comment by B3nz01d** [comment]
Completed during Sprint 19
